sound/soc/sof/ipc.c

```diff
@@ -6,6 +6,9 @@
  * Copyright(c) 2017 Intel Corporation. All rights reserved.
```

Comment:
checkpatch issues:
WARNING: 'mesage' may be misspelled - perhaps 'message'?
#92: FILE: sound/soc/sof/ipc.c:158:
+	/* add mesage to transmit list */

WARNING: 'mesage' may be misspelled - perhaps 'message'?
#113: FILE: sound/soc/sof/ipc.c:182:
+	/* send message if HW read and mesage in TX list */

CHECK: Please don't use multiple blank lines
#267: FILE: sound/soc/sof/ipc.c:366:
+
+

WARNING: 'postion' may be misspelled - perhaps 'position'?
#453: FILE: sound/soc/sof/ipc.c:478:
+/* get stream postion IPC - use faster MMIO method if available on platform */